### PR TITLE
i18(fr): Updated `configuration.md`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Updated `configuration.md` based on these 2 commits 31b8a5aed2bca363c1b05c683b020e596b70bf4a & e95167174e3eab3790328b8e42517abcbca04ff3

### CODE REVIEW

File: docs/src/content/docs/fr/reference/configuration.md [hunk 1]
```diff
@@ -378,3 +379,35 @@ la date basée sur Git en utilisant [le champ `lastUpdated` du frontmatter](/fr/
...
+
+Définnissez le chemin de l'icône par défaut pour votre site Web qui doit être situé dans le répertoire `public/` et être un fichier d'icône valide (`.ico`, `.gif`, `.jpg`, `.png` ou `.svg`).
+
+```js
+starlight({
+  favicon: '/images/favicon.svg',
+}),
+```
+
+Si vous avez besoin de définir des variantes supplémentaires ou des favicons de secours, vous pouvez ajouter des balises en utilisant l'option [`head`](#head) :
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/reference/configuration.md:396]:
```suggestion
Si vous avez besoin de définir des variantes supplémentaires ou des icônes de secours, vous pouvez ajouter des balises en utilisant l'option [`head`](#head) :
```

File: docs/src/content/docs/fr/reference/configuration.md [hunk 2]
```diff
@@ -378,3 +379,35 @@ la date basée sur Git en utilisant [le champ `lastUpdated` du frontmatter](/fr/
...
+
+```js
+starlight({
+  favicon: '/images/favicon.svg'.
+  head: [
+    // Ajouter un favicon ICO de secours pour Safari.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/reference/configuration.md:402]:
```suggestion
    // Ajouter une icône ICO de secours pour Safari.
```

**Review by HiDeoo — COMMENTED** [review]
I think we might as well always use "icône" and avoid the anglicism but except this detail, looks good to me 👍

**Comment by dreyfus92** [comment]
Thanks for reviewing @HiDeoo 😁, I will try to avoid anglicisms at all cost while translating to french

**Review by delucis — APPROVED** [review]
Thank you @dreyfus92 and @HiDeoo!

**Pull request merged**